x-pack/plugins/endpoint/public/plugin.ts

```diff
@@ -26,6 +27,17 @@ export class EndpointPlugin
     > {
   public setup(_core: CoreSetup, plugins: EndpointPluginSetupDependencies) {
     const resolverEmbeddableFactory = new ResolverEmbeddableFactory();
+    _core.application.register({
```

Comment:
could you rename this to `core`. The `_` prefix is tsserver / tsc speak for 'this is an unused variable'

Reply:
got it, will do